content/press/press-content.md

```diff
@@ -1,32 +1,19 @@
 <!--This is the main content file to edit for this page. It is embedded in src/pages/press.js-->
 
-## Press: Contact environmentalenforcementwatch@gmail.com
+**Press Contact: environmentalenforcementwatch@gmail.com
 
 <!--Remove '<!--' and similar-looking closing tag below to uncomment when we have some coverage -->
 <!--# Press Coverage of Congressional Committee Reports-->
 
-## Press Coverage of COVID EPA Enforcement Report
+## EEW Reports
 
-IMAGE = REPORT COVER
+* Executive Summary – 2020 EEW Congressional Report Card Findings (no link yet)
```

Comment:
Exec summary, not the full report?

We could also hedge a bit with this and say "coming soon" so that it can be live on the site but not be linked